File: src/BukkitPE/inventory/Transaction.php
<?php
namespace BukkitPE\inventory;

use BukkitPE\item\Item;

interface Transaction{

	/**
	 * @return Inventory
	 */
	public function getInventory();

	/**
	 * @return int
	 */
	public function getSlot();

	/**
	 * @return Item
	 */
	public function getSourceItem();

	/**
	 * @return Item
	 */
	public function getTargetItem();

	/**
	 * @return float
	 */
	public function getCreationTime();
}

File: src/main/java/net/BukkitPE/inventory/Transaction.java
package net.BukkitPE.inventory;

import net.BukkitPE.item.Item;

/**

 * BukkitPE Project
 */
public interface Transaction {

    Inventory getInventory();

    int getSlot();

    Item getSourceItem();

    Item getTargetItem();

    long getCreationTime();
}
